### PR TITLE
Add fastboot tests to the build

### PR DESCRIPTION
**Pull request by maxfierke**
Adds a simple smoke-test for making sure `{{ember-tooltip}}` doesn't explode when used with Fastboot

### CODE REVIEW

File: fastboot-tests/index-test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/* eslint-env node, mocha */
+'use strict';
+
+const expect = require('chai').expect;
```
Comment (sir-dunxalot) [inline fastboot-tests/index-test.js]: Are these files not run through `babel`?

Comment (sir-dunxalot) [inline fastboot-tests/index-test.js]: Are these files not run through `babel`?

File: fastboot-tests/index-test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    return this.visit('/')
+      .then(function(res) {
+        let $ = res.jQuery;
+        let response = res.response;
+
+        // add your real tests here
+        expect(response.statusCode).to.equal(200);
+        expect($('body').length).to.equal(1);
+        expect($('h1').text().trim()).to.equal('ember-fastboot-addon-tests');
+        expect($('.ember-tooltip-base').text().trim()).to.contain('i am an inline tooltip');
```
Comment (sir-dunxalot) [inline fastboot-tests/index-test.js]: Are we able to use our own [test helpers](https://github.com/sir-dunxalot/ember-tooltips#test-helpers) here?

**Comment by sir-dunxalot** [comment]
Can you quickly explain the reasoning for using `chai`?

**Comment by maxfierke** [comment]
@sir-dunxalot pretty much all of this is just what was generated by `ember-fastboot-addon-tests`, so I'm not sure whether it supports either qunit or the test helpers (my guess is no, on the test helpers front, since those are probably not setup to work with simpledom)

This is passing locally and failing on CI for some unknown reason, so I'm going to try going the `ember-cli-fastboot-testing` route, instead.

**Comment by maxfierke** [comment]
closing in favor of #289

**Pull request closed without merging**